pyseir/deployment/model_to_observed_shim.py

```diff
@@ -1,54 +1,106 @@
-from typing import Sequence
-
 import numpy as np
 
 
-def shim_deaths_model_to_observations(
-    model_death_ts: Sequence, idx: int, observed_latest: dict, log
-):
+def strict_shim(model: float, observed: float, log) -> float:
     """
     Take model outputs and calculate a "shim" value that can be added to the entire modeled
     cumulative deaths series to make them match the latest (i.e. today's) actual cumulative deaths.
     As a consequence, this will shift the future values by the same amount.
 
     Parameters
     ----------
-    model_death_ts
-        Model array sequence for cumulative deaths
-    idx
-        Index on which to align
-    observed_latest
-        Dictionary of latest values taken from combined dataset for the fips of interest.
+    model
+        model estimate for latest date
+    observed
+        observed value for latest date
     log
         Log Instance
     Return
     ------
-    shimmed_death: float
+    shim: float
         Value to shim the timeseries by
     """
-    observed_latest_cum_deaths = observed_latest["deaths"]
 
     # There are inconsistent None/"NaN" -> force all to np.nan for this scope.
-    if observed_latest_cum_deaths is None:
-        observed_latest_cum_deaths = np.nan
+    if observed is None:
+        observed = np.nan
 
-    model_latest_cum_deaths = model_death_ts[idx]
+    if np.isnan(observed):
+        shim = 0
+    elif observed == 0:
+        # As of 19 June 2020, the observed dataset is still being validated for erroneous inputs.
```

Comment:
@ghop02 We should QA these cases and see if this is still true for build latest. I would like to remove these special cases.

Reply:
When you say "observed dataset" which dataset is that referring to?

Reply:
I use "observed" to mean ground truth data, which in this case is whatever is returned from latest combined_dataset.